Installation/include/CGAL/version_macros.h

```diff
@@ -33,6 +33,11 @@
 #define CGAL_STR_STR(X) #X
 #endif
 
+#ifndef CGAL_str
+#define CGAL_xstr(s) #s
+#define CGAL_str(s) CGAL_xstr(s)
+#endif
```

Comment:
Those macros are not used anywhere, and are identical to `CGAL_STR` and `CGAL_STR_STR` defined just a few lines above.

Reply:
I just copy pasted what the previous release creation script was adding in `version.h`

Reply:
:-)